cmd/soroban-rpc/main.go

```diff
@@ -267,8 +266,7 @@ func main() {
 			serviceConfig.LedgerEntryStorageTimeout = time.Duration(ledgerEntryStorageTimeoutMinutes) * time.Minute
 			serviceConfig.CoreRequestTimeout = time.Duration(coreTimeoutSeconds) * time.Second
 			serviceConfig.MaxHealthyLedgerLatency = time.Duration(maxHealthyLedgerLatencySeconds) * time.Second
-			exitCode := daemon.Run(serviceConfig, endpoint)
-			os.Exit(exitCode)
+			daemon.Run(serviceConfig, endpoint)
```

Comment:
not sure why you'd ignoring the deamon.Run return.. it could be useful to let the Deamon return an error code.

Reply:
We can add it when/if needed, right now the daemon just starts  the webserver, which returns nothing

Reply:
yes, I just reviewed `support.http.Run`; admittedly, I'm not a big fan of that - I'd rather having a single location ( i.e. main function ) have all the os.Exit calls.
But I agree that at this point it's not needed.